specification/keyvault/resource-manager/Microsoft.KeyVault/preview/2020-04-01-preview/managedHsm.json

```diff
@@ -0,0 +1,634 @@
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Get information about all managed HSM Pools in the specified subscription.",
+            "schema": {
+              "$ref": "#/definitions/ManagedHsmListResult"
+            }
+          },
+          "default": {
+            "description": "The error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ManagedHsmError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "List managed HSM Pools in a subscription": {
+            "$ref": "./examples/ManagedHsm_ListBySubscription.json"
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    }
+  },
+  "definitions": {
+    "ManagedHsmSku": {
+      "properties": {
+        "family": {
+          "type": "string",
+          "description": "SKU Family of the managed HSM Pool",
+          "enum": [
+            "B"
+          ],
+          "x-ms-enum": {
+            "name": "ManagedHsmSkuFamily",
+            "modelAsString": true
+          }
+        },
+        "name": {
+          "type": "string",
+          "description": "SKU of the managed HSM Pool",
+          "enum": [
+            "Standard_B1",
+            "Custom_B32"
+          ],
+          "x-ms-enum": {
+            "name": "ManagedHsmSkuName",
+            "modelAsString": false
+          }
+        }
+      },
+      "description": "SKU details",
+      "required": [
+        "name",
+        "family"
+      ]
+    },
+    "ManagedHsmProperties": {
+      "properties": {
+        "tenantId": {
+          "type": "string",
+          "format": "uuid",
+          "description": "The Azure Active Directory tenant ID that should be used for authenticating requests to the managed HSM pool."
+        },
+        "securityDomainId": {
+          "type": "string",
+          "format": "uuid",
+          "description": "The security domain id that should be used for restoring data from this managed HSM pool to a new managed hsm pool."
+        },
+        "initialAdminObjectIds": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "Array of initial administrators object ids for this managed hsm pool."
+        },
+        "hsmPoolUri": {
+          "type": "string",
+          "description": "The URI of the managed hsm pool for performing operations on keys and secrets."
+        },
+        "enableSoftDelete": {
+          "type": "boolean",
+          "default": true,
+          "description": "Property to specify whether the 'soft delete' functionality is enabled for this managed HSM pool. If it's not set to any value(true or false) when creating new key vault, it will be set to true by default. Once set to true, it cannot be reverted to false."
```

Comment:
Based on Amit's recent update to the doc, i think we wont give an option to set enableSoftDelete = false.